src/main/pages/che-7/end-user-guide/ref_devfile-reference.adoc

```diff
@@ -386,9 +395,9 @@ components:
 
 [NOTE]
 ====
-* The variable expansion works between the environment variables and it uses the Kubernetes convention for the variable references.
+* The variable expansion works between the environment variables, and it uses the Kubernetes convention for the variable references.
 
-* You can use the predefined variables in your own definitions.
+* Use the predefined variables in your own definitions.
```

Comment:
Maybe (I am not sure) `One may use` is better, because it shows ability, but do not 'force' a user to use predefined variables.

Reply:
Thanks for clarifying.